src/constants.h

```diff
@@ -55,11 +55,14 @@ constexpr char kOpenVINORuntimeBackend[] = "openvino";
 
 constexpr char kPyTorchLibTorchPlatform[] = "pytorch_libtorch";
 constexpr char kPyTorchLibTorchFilename[] = "model.pt";
+constexpr char kPyTorchPythonPlatform[] = "pytorch_python";
 constexpr char kPyTorchBackend[] = "pytorch";
 
 constexpr char kPythonFilename[] = "model.py";
 constexpr char kPythonBackend[] = "python";
 
+constexpr char kVLLMBackend[] = "vllm";
```

Comment:
Do we need these for any new python based backend? Would be good to avoid that if possible - we shouldn't need to update the constants if someone adds a backend in the future

Reply:
We do not need them. `constexpr char kVLLMBackend[] = "vllm";` is removed, and they are handled as custom backend.